cli-output/src/cli_output.rs

```diff
@@ -1216,6 +1239,8 @@ pub struct CliStakeState {
     pub deactivating_stake: Option<u64>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub epoch_rewards: Option<Vec<CliEpochReward>>,
+    #[serde(skip_serializing)]
```

Comment:
🤔 why only skip serialization, but not deserialization?

Reply:
I can answer that: because we don't ever use these Cli output structs for deserialization of any kind. Arguably lazy, but we have no use-case and there's never been any external demand for it.

Reply:
Yeah that makes sense to me, I didn't really expect we'd be deserializing them; yet we derive `Deserialize`.

Eh, I'm fine to leave as is. Atleast it's just consistent with the other code, and not a super compelling reason to remove Deserialize except maybe slightly better compile times.

Reply:
Yeah, true. We could play with removing Deserialize sometime.